.agents/skills/adding-a-dsl-feature/SKILL.md

```
@@ -0,0 +1,44 @@
+---
+name: adding-a-dsl-feature
+description: Use when implementing any new DSL feature or picking up a feature issue in KyoriAdventureDSL — the end-to-end, small-slice workflow from issue to merge.
+---
+
+# Adding a DSL feature
+
+A repeatable, **small-slice** workflow. The goal is one focused, well-tested, idiomatic change — not a kitchen sink.
+
+## 1. Scope from the issue
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)